packages/core/lib/send-result/node-http.ts

```diff
@@ -15,10 +15,12 @@ export async function sendResponseResult(
   for (const { name, value } of responseResult.headers) {
     rawResponse.setHeader(name, value);
   }
+  const data = Buffer.from(JSON.stringify(transformResult(responseResult.payload)))
```

Comment:
Not sure if Buffer.from is needed. Other than that, it looks good.

Reply:
I first tried with just length of the `JSON.stringify` output, but it doesn't cover the length of special characters and length was incorrect.
Now that you mention it maybe we could use `Buffer.byteLength` function instead so that we don't reserve extra space for it. I'll change it to use that.